src/App.scss

```diff
@@ -0,0 +1,281 @@
+@import './style/scss/import';
+
+* {
+  box-sizing: border-box;
+  font-family: $primary-font;
+}
+
+.btn {
+  font-size: 14px;
+  font-weight: 700;
+  color: $primary-color;
+  padding: 8px 1rem;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 4.375rem;
+  margin-top: 3.4375rem;
+  justify-content: space-between;
+  .headerListing {
+    display: flex;
+    align-items: center;
+    .svgwrapper {
+      width: 70px;
+      height: 26px;
+      svg {
+        height: 100%;
+        width: 100%;
+      }
+    }
+    ul {
+      display: flex;
+      justify-content: space-between;
+      list-style-type: none;
+      margin-left: 7.5rem;
+      margin-bottom: 0;
+      & :not(:last-child) {
+        margin-right: 3.125rem;
+      }
+      li {
+        a {
+          font-size: 14px;
+          font-weight: 600;
+          color: $primary-font-color;
+          &:hover,&:active {
+            color: $primary-color;
+          }
+        }
+      }
+    }
+  }
+  .headerBtn {
+    .signUpBtn {
+      border: 1px solid $primary-color;
+      border-radius: 0%;
+    }
+  }
+}
+
+.intro {
+  text-align: center;
+  margin: 6.25rem auto 0;
+  h1 {
+    margin: 0 auto;
+    max-width: 840px;
+  }
+  p {
+    font-size: 1.125rem;
+    margin: 2.5rem auto 0;
+    max-width: 580px;
+  }
+}
+
+.inputField {
+  display: flex;
+  justify-content: center;
+  margin-top: 3.5rem;
+  input {
+    font-size: 1rem;
+    max-width: 350px;
```

Comment:
first create input wrapper than move max-width to input-wrapper and give width: 100% to input